src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,156 @@
+                    property string description: VPNl18n.TipsAndTricksAdvancedTipsDescription
+                    property var filter: (addon) => addon.advanced && !addon.highlighted
+                }
+            }
+
+            ColumnLayout {
+                id: layoutAll
+
+                anchors.fill: parent
+                anchors.topMargin: VPNTheme.theme.vSpacing
+                anchors.bottomMargin: VPNTheme.theme.vSpacing
+                anchors.leftMargin: VPNTheme.theme.windowMargin
+                anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                VPNTutorialList {
+                    parentWidth: layoutAll.width
+                    customTutorialFilter: (addon) => addon.highlighted
+                }
+
+                Repeater {
+                    model: tipsAndTricksSections.count
+
+                    Column {
+                        property var section: tipsAndTricksSections.get(index)
+
+                        visible: guidesList.count || tutorialsList.count
+
+                        Layout.alignment: Qt.AlignTop
+                        Layout.topMargin: 32
+                        spacing: VPNTheme.theme.vSpacingSmall
+
+                        VPNBoldLabel {
+                            text: section.title
+
+                            Accessible.role: Accessible.StaticText
+                            Accessible.name: text
+                        }
+
+                        VPNTextBlock {
+                            text: section.description
+
+                            Accessible.role: Accessible.StaticText
+                            Accessible.name: text
+                        }
+
+                        VPNGuideList {
+                            id: guidesList
+                            parentWidth: layoutAll.width
+                            customGuideFilter: section.filter
+                        }
+
+                        VPNTutorialList {
+                            id: tutorialsList
+                            parentWidth: layoutAll.width
+                            customTutorialFilter: section.filter
+                            showVPNTour: false
+                        }
+                    }
+                }
+            }
+        },
+
+        // Tutorials
+        VPNFlickable {
+            flickContentHeight: layoutTutorial.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+            interactive: flickContentHeight > height
+
+            ColumnLayout {
+                id: layoutTutorial
+
+                anchors.fill: parent
+                anchors.topMargin: VPNTheme.theme.vSpacing
+                anchors.bottomMargin: VPNTheme.theme.vSpacing
+                anchors.leftMargin: VPNTheme.theme.windowMargin
+                anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                VPNTutorialList {
+                    parentWidth: layoutTutorial.width
+                }
+            }
+        },
+
+        // Tips
```

Comment:
Looking at the mocks, it seems that the filtered tips section should also have the "quick" and "advanced" section headers. Was there a design change here?

Reply:
Which mocks are you referring to? I guess there must have been a deisgn change, because the spec that I got does not include the headers https://www.figma.com/file/LmWggJlWXCi23jOpitIDnA/VPN-Tips-%26-Tricks?node-id=988%3A37156 🤔

Reply:
...Nevermind, you are right. Changing this.